console/src/main/webapp/manager/app/components/role/role.es6

```diff
@@ -19,14 +19,21 @@ class RoleController {
     translate('role.deleteError', this.i18n)
 
     this.role = $injector.get('Role').get({id: $routeParams.role})
```

Comment:
you can use a function as a second argument to get to avoid storing `orginalID` in the controller

Reply:
Done with https://github.com/georchestra/georchestra/pull/2025/commits/0b241d87c7e16e53f2f68ee91cdb8d66ced72f61